src/models/meta.ts

```diff
@@ -30,6 +30,17 @@ if ((config as any).description) {
 		}
 	});
 }
+if ((config as any).url) {
+	Meta.findOne({}).then(m => {
+		if (m != null && m.uri == null) {
+			Meta.update({}, {
+				$set: {
+					uri: (config as any).url
+				}
+			});
+		}
+	});
+}
```

Comment:
Probably this section is unnecessary.
This section to be used only by settings that need to be migrated from file to DB.